packages/gatsby-theme-docs/src/layouts/internals/page-content-inset.js

```diff
@@ -1,9 +1,16 @@
 import styled from '@emotion/styled';
-import { designSystem } from '@commercetools-docs/ui-kit';
+import { designSystem, Markdown } from '@commercetools-docs/ui-kit';
 
 const PageContentInset = styled.div`
```

Comment:
Nit: can you maybe leave some comments explaining the reasons for the `section` selectors? So that we know in the future what the reason for that is. Thanks